app/assets/javascripts/backbone/views/loan_questions_view.coffee

```diff
@@ -0,0 +1,127 @@
+class MS.Views.LoanQuestionsView extends Backbone.View
```

Comment:
Should probably call this ManageLoanQuestionsView since there is also LoanQuestionnairesView